keyboards/DT40.c

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2021 linlin

```

Comment:
This file should not exist at this location.

Reply:
Sorry, I'm a novice. Thank you for your guidance and tips.